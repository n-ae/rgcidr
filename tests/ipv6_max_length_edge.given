ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
0000:0000:0000:0000:0000:0000:0000:0000
ffff:0000:0000:0000:0000:0000:0000:0001
ffff:0000:0000:0000:0000:0000:0000:0000
0001:0000:0000:0000:0000:0000:0000:0000
0000:0001:0000:0000:0000:0000:0000:0000
0000:0000:0001:0000:0000:0000:0000:0000
ffff::1
ffff::
::1
::
1::
::ffff:ffff
ffff::ffff
1::ffff:ffff:ffff:ffff
ffff:ffff:ffff:ffff::1
ffff:ffff:ffff:ffff::
::ffff:ffff:ffff:ffff
f::f::f
ffff::ffff::ffff
:::
