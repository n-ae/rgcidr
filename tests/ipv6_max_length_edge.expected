ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff
ffff:0000:0000:0000:0000:0000:0000:0001
ffff:0000:0000:0000:0000:0000:0000:0000
ffff::1
ffff::
ffff::ffff
ffff:ffff:ffff:ffff::1
ffff:ffff:ffff:ffff::
